xml/art-nvidia-vgpu.xml

```diff
@@ -819,39 +819,80 @@ GPU 0: NVIDIA A100-PCIE-40GB (UUID: GPU-ee14e29d-dd5b-2e8e-eeaf-9d3debd10788)
     </para>
    </step>
    <step>
-    <para>
-     Update the following options:
-    </para>
-    <variablelist>
-     <varlistentry>
-      <term>ServerAddress</term>
-      <listitem>
-       <para>
-        Add your license server IP address.
-       </para>
-      </listitem>
-     </varlistentry>
-     <varlistentry>
-      <term>ServerPort</term>
-      <listitem>
-       <para>
-        Use the default "7070" or port configured during the server setup.
-       </para>
-      </listitem>
-     </varlistentry>
-     <varlistentry>
-      <term>FeatureType</term>
-      <listitem>
-       <para>
-        For GPU passthrough, set <option>FeatureType</option> to
-        <literal>4</literal> for compute and <literal>2</literal> for graphic
-        purpose. For virtual GPU case, whatever vGPU type is created via
-        <command>mdev</command> determines the feature set that is enabled in
-        &vmguest;.
-       </para>
-      </listitem>
-     </varlistentry>
-    </variablelist>
+    <substeps>
+     <step>
+      <para>
+       For licenses that are served from the &nvidia; License System,
+       update the following options:
+      </para>
+      <variablelist>
+       <varlistentry>
+        <term>FeatureType</term>
+        <listitem>
+         <para>
+          For GPU passthrough, set <option>FeatureType</option> to
+          <literal>4</literal> for compute and <literal>2</literal> for graphic
+          purpose. For virtual GPU case, whatever vGPU type is created via
+          <command>mdev</command> determines the feature set that is enabled in
```

Comment:
```suggestion
          For GPU passthrough, set <option>FeatureType</option> to
          <literal>4</literal> for computing and <literal>2</literal> for graphic
          purposes. In case of a virtual GPU, whatever vGPU type is created via
          <command>mdev</command> determines the feature set that is enabled in
```